modules/ducktests/tests/ignitetest/services/utils/ignite_aware.py

```diff
@@ -200,16 +200,19 @@ def __exec_on_node(node, task, start_waiter=None, delay_ms=0, time_holder=None):
 
         task(node)
 
-    def disconnect(self, nodes=None):
+    def drop_network(self, nodes=None):
         """
         Disconnects node from cluster.
         """
-        self.logger.info("Disconnecting the node.")
+        self.logger.info("Disconnecting the node(s)...")
 
         if nodes is None:
             assert self.num_nodes == 1
             nodes = self.nodes
 
+        for node in nodes:
+            self.logger.debug("Disconnecting " + node.account.hostname + ".")
```

Comment:
I suppose we would need this at info level. Next code provides on debug "Activating netfilter on...". This also says which node ig going to get stopped. But with extented netfilter info.